src/main/java/fr/sncf/osrd/infra/trackgraph/TrackSection.java

```diff
@@ -172,19 +173,11 @@ public double position(EdgeDirection direction, double pos) {
             throw new InvalidInfraException(String.format("invalid PointSequence end for %s", id));
     }
 
-    private <ValueT> void validateRanges(RangeSequence<ValueT> ranges) throws InvalidInfraException {
-        if (ranges.getFirstPosition() < 0.)
-            throw new InvalidInfraException(String.format("invalid RangeSequence start for %s", id));
-        if (ranges.getLastPosition() >= length)
-            throw new InvalidInfraException(String.format("invalid RangeSequence end for %s", id));
-    }
-
     /**
      * Ensure the edge data in consistent.
      * @throws InvalidInfraException when discrepancies are detected
      */
-    public void validate() throws InvalidInfraException {
-        validateRanges(slope);
+    public void validate() {
         // TODO: validate speed limits
```

Comment:
I'm not sure what this TODO means. This function is empty so it might be deleted.